### ISSUE TITLE
[dropdown] is corrupting data (/w multiple enabled & allowAddition)

### ISSUE THREAD

**Issue by mvorisek** [fomantic/Fomantic-UI#2246]
# Bug Report

## Steps to reproduce

https://jsfiddle.net/e8o3psvr/

## Expected result

```
[{
  name: "recipient",
  value: "Username &lt;user@emaildomain.de&gt"
}]
[{
  name: "recipient",
  value: "Username &lt;user@emaildomain.de&gt"
}]
[{
  name: "recipient",
  value: "Username Xuser@emaildomain.deX"
}]
```

## Actual result

```
[{
  name: "recipient",
  value: "Username &lt;user@emaildomain.de&gt;,Username &amp;lt;user@emaildomain.de&amp;gt;"
}]
[{
  name: "recipient",
  value: "Username &lt;user@emaildomain.de&gt;,Username &amp;lt;user@emaildomain.de&amp;gt;"
}]
[{
  name: "recipient",
  value: "Username Xuser@emaildomain.deX"
}]
```

## Version
2.8.8

Also and probably related, discovered in https://github.com/fomantic/Fomantic-UI/pull/2247#issuecomment-1052781930, `xx<yy` input produces very strange result.

**Comment by mvorisek** [comment]
What is the reason of this behaviour and is there any workaround?

**Comment by mvorisek** [comment]
Is this a bug or is the value expected to be encoded twice, eg. once for the value attribute and once for each listed value?

**Comment by mvorisek** [comment]
This issue is quite dangerous as it corrupts data. Can it be fixed soon and is there any easy fix from JS when using the latest v2.8.8 libs?

**Comment by lubber-de** [comment]
I am working on it already. PR is on its way. The only workaround is to already htmlentity encode the predefined values.

**Comment by mkrecek234** [comment]
@lubber-de Any news on a fix for this issue? Would be welcomed a lot. Thank you

**Comment by mkrecek234** [comment]
@lubber-de We are approaching our next release using Fomantic-UI - would be great if you could fix this issue? Any chance?

**Comment by lubber-de** [comment]
This is the last remaining bug for 2.9.0, so....yes... 😉

**Comment by lubber-de** [comment]
Fixed by #2474 
See your adjusted jsfiddle here https://jsfiddle.net/lubber/61u0xpL8/5
In your use case you now have to use `preserveHTML:false` to make the < > being rendered inside the label. However the values are not encoded and not added twice anymore
